app/experimenter/experiments/tests/api/v5/test_serializers.py

```diff
@@ -233,6 +233,59 @@ def test_branch_name_cant_slugify(self):
         )
 
 
+class TestNimbusExperimentDocumentationLinkMixin(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.user = UserFactory()
+
+    def test_serializer_update_links(self):
```

Comment:
I would also check in here that the previously stored links are deleted.

Reply:
Adding a `self.assertEqual(len(links_data), len(documentation_links))` which I think should do the trick, since the contents are also asserted

Reply:
Actually, never mind that, just going to switch to a set assertion here and also over in `test_mutation`